worker/src/RTC/RTCP/CompoundPacket.cpp

```diff
@@ -8,23 +8,21 @@ namespace RTC
 {
 	namespace RTCP
 	{
-		thread_local static Utils::ObjectPool<CompoundPacket> CompoundPacketPool;
-
 		/* Instance methods. */
 
 		CompoundPacket::UniquePtr CompoundPacket::Create()
 		{
-			auto* packet = CompoundPacketPool.Allocate();
-
-			return UniquePtr(new (packet) CompoundPacket());
+			auto* packet = CompoundPacket::Allocator::Pool.Allocate();
+			CompoundPacket::Allocator::Pool.construct(packet);
```

Comment:
`construct`/`destroy` were deprecated in C++17 and removed in C++20. Do you know why? I'd prefer to use future-proof syntax in here if possible.

Reply:
It seems they have been removed for being duplicated to `std::allocator_traits`

http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2016/p0174r0.html#2.4

Reply:
Looks like ```std::allocator_traits``` available from C++11. I'll prepare commit with change